lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -503,6 +510,9 @@ def _stop_and_delete_vm_flags(
     ) -> None:
         node_context = get_node_context(node)
 
+        watchdog = Timer(60.0, self._watchdog_callback)
```

Comment:
Does it work like timeout? LISA uses the `func_timeout` package to manage timeout. Search for `func_set_timeout` and `FunctionTimeOut` about how it's used.

Reply:
I doubt the problem I am seeing can be recovered from, due to the use of shared/global resources in libvirt. Also, I want to see the callstacks of all threads, in case it reveals the cause of the deadlock. So, using `func_timeout` probably doesn't make sense here.